File: web/src/utils/random.ts
export class Random {
  public static element<T>(array: T[]): T {
    return array[Math.floor(Math.random() * array.length)];
  }

  public static between(min: number, max: number): number {
    return Math.floor(Math.random() * (max - min + 1)) + min;
  }
}

File: server/src/test/java/com/github/throyer/example/api/utils/Random.java
package com.github.throyer.example.api.utils;

import static java.lang.String.format;

import java.util.List;
import java.util.Locale;

import org.springframework.stereotype.Component;

import com.github.javafaker.Faker;

@Component
public class Random {  
  private static final java.util.Random JAVA_RANDOM = new java.util.Random();
  public static final Faker FAKER = new Faker(new Locale("pt", "BR"));

  public static Integer between(Integer min, Integer max) {
    return JAVA_RANDOM.nextInt(max - min) + min;
  }

  public static <T> T element(List<T> list) {
    return list.get(JAVA_RANDOM.nextInt(list.size()));
  }

  public static String code() {
    return format("%s%s%s%s", between(0, 9), between(0, 9), between(0, 9), between(0, 9));
  }
}